bpf/ingress_node_firewall.h

```diff
@@ -44,10 +44,6 @@ struct ruleStatistics_st {
         __u64 packets;
         __u64 bytes;
     } deny_stats;
-    struct nomatch_stats_st {
```

Comment:
why remove this?

Reply:
stats key ruleID now this action takes place only if there is no rule hit at all so I don't have rule ID to add those stats on
also I am using ruleId 0 to catch global no match drop /CIDR with new trace